ci/integration_tests/src/helpers/chain.rs

```diff
@@ -3,7 +3,6 @@ use cosm_orc::orchestrator::cosm_orc::CosmOrc;
 use cosm_orc::{
     config::cfg::Config, config::key::SigningKey, profilers::gas_profiler::GasProfiler,
 };
-use ctor::ctor;
```

Comment:
We really dont need this dependency, I just did the same ctor annotation functionality with `get_or_init()`.